releases/net472/KnownIssues/706819 - Winforms applications may have problems rendering UI on latest Operating Systems .md

```diff
@@ -0,0 +1,11 @@
+ # Winforms application may have problems rendering application UI and instead, may render a white screen with a red ‘X’ mark across.
```

Comment:
- There are a number of occurrences of "Winforms". They should all be changed to "WinForms" 
- Winforms application --> WinForms applications 
- application UI --> the application UI
- no comma after "instead"
- across is a preposition that requires an object. Perhaps "a white screen with a large 'X' symbol"?